src/memory/app.js

```diff
@@ -0,0 +1,45 @@
+let cards = []
+
+for (let i = 0; i < 8; i++) {
+    let card = `C:\Images\\${i}.png`
```

Comment:
Ne bi trebalo da koristiš putanje iz fajlsistema. Pokreni server i koristi http ili koristi relativne putanje. Ovo npr. neće raditi ako pokrenem kod mene na kompu.

Reply:
To sam i radio prvo, jer dok sam radio u React iz lokala nisam ni mogao da pokrenem zbog onih security ogranicenja. Vraticu na to